package.json

```diff
@@ -14,7 +16,7 @@
   "author": "maisymoe",
   "license": "BSD-3-Clause",
   "devDependencies": {
-    "@biomejs/biome": "^1.5.3",
+    "@biomejs/biome": "1.5.3",
```

Comment:
Do we need to limit this to specifically `1.5.3`? `^1.5.3` allows us to still get feature updates, but breaking changes don't count.